Conda CI failing on main: ImportError: cannot import name 'url_quote' from 'werkzeug.urls'

- https://github.com/scitt-community/scitt-api-emulator/actions/runs/6461497669/job/17541274650#step:4:50

```console
$ python -m pytest
============================= test session starts ==============================
platform linux -- Python 3.8.16[pypy-7.3.11-final], pytest-7.2.1, pluggy-1.3.0
rootdir: /home/runner/work/scitt-api-emulator/scitt-api-emulator
plugins: anyio-4.0.0
collected 0 items / 2 errors

==================================== ERRORS ====================================
______________________ ERROR collecting tests/test_cli.py ______________________
ImportError while importing test module '/home/runner/work/scitt-api-emulator/scitt-api-emulator/tests/test_cli.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/share/miniconda/envs/scitt/lib/pypy3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_cli.py:7: in <module>
    from scitt_emulator import cli, server
scitt_emulator/cli.py:6: in <module>
    from scitt_emulator import client, server
scitt_emulator/server.py:9: in <module>
    from flask import Flask, request, send_file, make_response
/usr/share/miniconda/envs/scitt/lib/pypy3.8/site-packages/flask/__init__.py:5: in <module>
    from .app import Flask as Flask
/usr/share/miniconda/envs/scitt/lib/pypy3.8/site-packages/flask/app.py:30: in <module>
    from werkzeug.urls import url_quote
E   ImportError: cannot import name 'url_quote' from 'werkzeug.urls' (/usr/share/miniconda/envs/scitt/lib/pypy3.8/site-packages/werkzeug/urls.py)
_____________________ ERROR collecting tests/test_docs.py ______________________
ImportError while importing test module '/home/runner/work/scitt-api-emulator/scitt-api-emulator/tests/test_docs.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/share/miniconda/envs/scitt/lib/pypy3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_docs.py:24: in <module>
    from .test_cli import (
tests/test_cli.py:7: in <module>
    from scitt_emulator import cli, server
scitt_emulator/cli.py:6: in <module>
    from scitt_emulator import client, server
scitt_emulator/server.py:9: in <module>
    from flask import Flask, request, send_file, make_response
/usr/share/miniconda/envs/scitt/lib/pypy3.8/site-packages/flask/__init__.py:5: in <module>
    from .app import Flask as Flask
/usr/share/miniconda/envs/scitt/lib/pypy3.8/site-packages/flask/app.py:30: in <module>
    from werkzeug.urls import url_quote
E   ImportError: cannot import name 'url_quote' from 'werkzeug.urls' (/usr/share/miniconda/envs/scitt/lib/pypy3.8/site-packages/werkzeug/urls.py)
=============================== warnings summary ===============================
../../../../../usr/share/miniconda/envs/scitt/lib/pypy3.8/site-packages/requests_toolbelt/_compat.py:56
  /usr/share/miniconda/envs/scitt/lib/pypy3.8/site-packages/requests_toolbelt/_compat.py:56: DeprecationWarning: 'urllib3.contrib.pyopenssl' module is deprecated and will be removed in a future release of urllib3 2.x. Read more in this issue: https://github.com/urllib3/urllib3/issues/2680
    from requests.packages.urllib3.contrib.pyopenssl \

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR tests/test_cli.py
ERROR tests/test_docs.py
!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!
========================= 1 warning, 2 errors in 1.81s =========================
Error: Process completed with exit code 2.
```

--- a/environment.yml
+++ b/environment.yml
@@ -38,3 +38,4 @@ dependencies:
     - jsonschema==4.17.3
     - jwcrypto==1.5.0
     - PyJWT==2.8.0
+    - werkzeug==2.2.2

